sites/partners/lib/helpers.ts

```diff
@@ -162,7 +162,18 @@ export const createTime = (
  */
 export const createDate = (formDate: { year: string; month: string; day: string }) => {
   if (!formDate || !formDate?.year || !formDate?.month || !formDate?.day) return null
-  return new Date(`${formDate.month}-${formDate.day}-${formDate.year}`)
+  const tzOffset = new Date().getTimezoneOffset() / -60
+  let tzString: string
+
+  if (tzOffset >= 0) {
+    if (tzOffset < 10) tzString = `+0${tzOffset}:00`
+    if (tzOffset >= 10) tzString = `+${tzOffset}:00`
+  } else {
+    if (tzOffset > -10) tzString = `-0${tzOffset.toString().substring(1)}:00`
+    if (tzOffset <= -10) tzString = `${tzOffset}:00`
+  }
+
+  return new Date(`${formDate.year}-${formDate.month}-${formDate.day}T00:00:00.000${tzString}`)
```

Comment:
Can we use dayjs to handle some of this, like setting the utc offset. I know adding dayjs is in another PR, but using that could probably take some logic out of our hands.

Reply:
Agreed, I worked on this before working on the other brach. Up to you if you want to merge this as-is or redo after adding day.js

Reply:
@seanmalbert All right, the dayjs stuff is merged in now, so I'll work on updating this and getting rid of so much custom logic.